plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -129,7 +130,7 @@ open class DefaultPageCreator(
         divergentBlock("Types", types, ContentKind.Classlikes, extra = mainExtra + SimpleAttr.header("Types"))
         divergentBlock(
             "Functions",
-            s.functions,
+            s.functions.sort(),
```

Comment:
Just a question: 
Would we maybe benefit from (whoever is containing this functions) already having the ordered the same way?
Maybe we are using the functions somewhere else and have unexpected unintuitive order somewhere? 🤔

Reply:
I agree and that was my first idea, but then after a discussion with @Kordyjan we decided to do it this way, as the presentation side is responsible for the correct displaying of declarations (and TBH it's faster this way as we don't have to modify the data structure)

Reply:
Thats okay ☺️ Good argument!